lib/mixins/listen-on-root.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Issue #569: collapse::toggle::state triggered too many times
+ * @link https://github.com/bootstrap-vue/bootstrap-vue/issues/569
+ */
+export default {
+    data() {
```

Comment:
I don't believe private props are non-reactive [anymore](https://github.com/vuejs/Discussion/issues/353), but perhaps private notation is best.